### PR TITLE
test(e2e): add cf import s3

### PR DESCRIPTION
**Pull request by iamhopaul123**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: e2e/cloudfront/cloudfront_test.go [hunk 1]
```diff
@@ -98,6 +121,52 @@ var _ = Describe("CloudFront", func() {
...
+					parsed, err := arn.Parse(resource["physicalID"])
+					Expect(err).NotTo(HaveOccurred())
+					accountID = parsed.AccountID
+				case "CloudFrontDistribution":
+					cfDistID = resource["physicalID"]
+				}
+			}
+			Expect(accountID).ToNot(BeEmpty())
+			Expect(cfDistID).ToNot(BeEmpty())
+			_, err = s3Client.PutBucketPolicy(&s3.PutBucketPolicyInput{
```
Comment (efekarakus) [inline e2e/cloudfront/cloudfront_test.go:144]: I see, I assume we will add this piece of info into our documentation in the env manifest right?

File: e2e/cloudfront/cloudfront_test.go [hunk 2]
```diff
@@ -98,6 +121,52 @@ var _ = Describe("CloudFront", func() {
...
+			})
+			Expect(err).NotTo(HaveOccurred())
+			var accountID string
+			var cfDistID string
+			for _, resource := range out.Resources {
+				switch resource["logicalID"] {
+				case "PublicLoadBalancer":
+					parsed, err := arn.Parse(resource["physicalID"])
+					Expect(err).NotTo(HaveOccurred())
+					accountID = parsed.AccountID
```
Comment (efekarakus) [inline e2e/cloudfront/cloudfront_test.go:137]: lol that's a funny way of getting the account ID, is it not available in the physical ID of the CF Distribution?

**Review by efekarakus — APPROVED** [review]

File: e2e/cloudfront/cloudfront_test.go (hunk 1 above)
Comment (iamhopaul123) [reply e2e/cloudfront/cloudfront_test.go:144]: yeah it's in my doc pr.

File: e2e/cloudfront/cloudfront_test.go (hunk 2 above)
Comment (iamhopaul123) [reply e2e/cloudfront/cloudfront_test.go:137]: lol i wish the physical ID could be the full arn but no it is just the CF distribution ID.

**Review by huanjani — APPROVED** [review]
:shipit:

**Pull request merged**